backoffice-service/composer.json

```diff
@@ -75,7 +79,12 @@
     "extra": {
         "symfony": {
             "allow-contrib": false,
-            "require": "5.4.*"
+            "require": "^6.3"
```

Comment:
I guess if we require here 6.3, then we can't really pull `5.4`?
Let's require for every symfony component `6.3`

Reply:
OK, I've made that change.  It appears to work with Symfony 6.4 as well